nodejs/package.json

```diff
@@ -11,6 +11,8 @@
   "dependencies": {
     "body-parser": "1.15.2",
     "express": "4.14.0",
-    "smooch-core": "2.1.1"
+    "smooch-core": "2.1.1",
```

Comment:
Maybe I am misunderstanding but you'll need the newest version?
```suggestion
    "sunshine-conversations-client": "6.0.0",
```

Reply:
And same for the python dependencies in `python/requirements.txt`
`sunshine-conversations-client==6.0.0`

Reply:
For now I'm not including python, only js